clients/fides-js/docs/interfaces/Fides.md

```diff
@@ -32,24 +32,11 @@ existence of Fides *or* subscribe to the global `FidesInitialized` event (see
 </body>
 ```
 
-## Table of contents
```

Comment:
This TOC _was_ helpful, but I think it's OK. When we package these up and host on our docs site, the sidebar generates a new TOC from the headings anyways: https://ethyca.com/docs/dev-docs/js/reference/interfaces/Fides
<img width="1412" alt="image" src="https://github.com/ethyca/fides/assets/1834295/ef9f443a-b3db-4c2e-bd71-71cdba175a76">

Reply:
The inline TOC was annoying me for that reason. I think it's a good move. I like the sidebar much better.